harmonica/equivalent_layer/harmonic.py

```diff
@@ -74,13 +75,27 @@ class EQLHarmonic(vdb.BaseGridder):
         If None, will place one point source bellow each observation point at
         a fixed relative depth bellow the observation point [Cooper2000]_.
         Defaults to None.
-    relative_depth : float
-        Relative depth at which the point sources are placed beneath the
+    depth : float
+        Parameter used to control the depth at which the point sources will be
+        located.
+        If ``depth_type`` is equal to ``"relative"``, the ``depth`` specifies
+        the relative depth at which the point sources are placed beneath the
         observation points. Each source point will be set beneath each data
         point at a depth calculated as the elevation of the data point minus
-        this constant *relative_depth*. Use positive numbers (negative numbers
-        would mean point sources are above the data points). Ignored if
-        *points* is specified.
+        this *depth*. Use positive numbers (negative numbers would mean point
+        sources are above the data points).
+        If ``depth_type`` is equal to ``"constant"``, the ``depth`` specifies
+        the constant depth at which the point sources are placed beneath the
+        observation points. Every source point will be located at this *depth*.
+        Use positive numbers (negative numbers would mean point sources are
+        located above the zeroth level).
+        This parameter is ignored if *points* is specified.
+        Defaults to 500.
```

Comment:
It's so hard to choose a sensible default here. This is OK for now but we should try to think of a way to define this relative to some property of the input data (average spacing, etc).

Reply:
I agree that it's very hard to choose a default value for the `depth`. Nevertheless I've always saw it as a dummy value that people are encouraged to change and experiment with. I'm afraid to set a clever way to get it and people think that our  automated choice would be smarter than theirs.